conversion.go

```diff
@@ -98,6 +112,9 @@ func (z *Int) UnmarshalText(input []byte) error {
 // SetFromBig converts a big.Int to Int and sets the value to z.
 // TODO: Ensure we have sufficient testing, esp for negative bigints.
 func (z *Int) SetFromBig(b *big.Int) bool {
+	if b == nil {
+		return true
+	}
```

Comment:
Seems weird to `no-op` here. IMO we should either crash or _at least_ make sure to clear `z`. 

But I don't know, seems like the caller should do the `nil`-check.

Reply:
hm, i agree the caller should do the nil check.

regardless i think i will leave it in for convenience, and just make sure to clear z